build/images/ovs/apply-patches.sh

```diff
@@ -94,8 +94,8 @@ fi
 
 # This patch fixes a log file leak in OVS.
 # See https://github.com/antrea-io/antrea/issues/2003
-# It is fixed in the OVS master branch and will be included starting with OVS 2.18.
-if version_lt "$OVS_VERSION" "2.18.0" ; then
+# It is fixed in the OVS master branch and will be included starting with OVS 2.17.2.
+if version_lt "$OVS_VERSION" "2.17.2" ; then
```

Comment:
I checked 2.17.2 and 2.17.3 don't include the patch: https://github.com/openvswitch/ovs/commits/v2.17.3/lib/daemon-unix.c, why changing it? It would cause the regression of the log file leak.

Reply:
you're completely right, I saw `daemon-unix` and `leak` in the commit message but did not accurately check it.

thank you for catching this.